doc/zh-cn/plugins/node-status.md

```diff
@@ -0,0 +1,142 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+
+# 目录
+- [**插件简介**](#插件简介)
+- [**插件属性**](#插件属性)
+- [**插件接口**](#接口插件)
+- [**启用插件**](#启用插件)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+
+## 插件简介
+
+`node-status` 是APISIX的请求状态查询插件，返回基本的状态信息。
+
+
+## 插件属性
+
+插件属性为空。
+
+
+## 插件接口
+
+插件增加接口`/apisix/status`，可通过 [interceptors](plugin-interceptors.md)保护该接口。
+
+
+## 启用插件
+
+1. 配置文件apisix/conf/conf.yaml的plugins list中配置node-status
```

Comment:
The file name of `conf.yaml` is incorrect, it should be `config.yaml`, right?

Reply:
yes, I will change it. Thanks~